prac_06/languages.py

```diff
@@ -12,8 +12,12 @@ def main():
     visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
     print(python)
     languages = [python, ruby, visual_basic]
-    print(type(languages))
-    print(languages)
+    # print(type(languages))
+    # print(languages)
+    print("The dynamically typed languages are:")
+    for language in languages:
+        if language.typing == "Dynamic":
```

Comment:
Utilise the is_dynamic() method we created in ProgrammingLanguage class.

Reply:
That's right! I missed it. I changed the code now. Thanks!